roles/confluent.ksql/templates/ksql-server.properties.j2

```diff
@@ -34,14 +29,25 @@ sasl.jaas.config=com.sun.security.auth.module.Krb5LoginModule required \
    principal="{{ksql_kerberos_principal}}";
 {% endif %}
 
-# Monitoring
-confluent.monitoring.interceptor.security.protocol={{security_protocol}}
-{% if ssl_enabled|bool %}
-confluent.monitoring.interceptor.ssl.truststore.location=/var/ssl/private/client.truststore.jks
-confluent.monitoring.interceptor.ssl.truststore.password={{host_truststore_storepass}}
-confluent.monitoring.interceptor.ssl.keystore.location=/var/ssl/private/client.keystore.jks
-confluent.monitoring.interceptor.ssl.keystore.password={{host_keystore_storepass}}
-confluent.monitoring.interceptor.ssl.key.password={{host_cert_password}}
+{% for key, value in ksql.properties.items() %}
+{{key}}={{value}}
+{% endfor %}
+
+{% set schema_registries = groups.get('schema_registry', []) %}
+{% if schema_registries %}
+# Schema Registry Configuration
+ksql.schema.registry.url={% for host in groups['schema_registry'] %}{% if loop.index > 1%},{% endif %}{{ schema_registry_http_protocol }}://{{ host }}:{{ schema_registry_listener_port }}{% endfor %}
+
+{% endif %}
+
+# Monitoring Configuration
+confluent.monitoring.interceptor.security.protocol={{kafka_broker_security_protocol}}
+{% if kafka_broker_ssl_enabled|bool %}
+confluent.monitoring.interceptor.ssl.truststore.location={{truststore_path}}
+confluent.monitoring.interceptor.ssl.truststore.password={{truststore_storepass}}
+confluent.monitoring.interceptor.ssl.keystore.location={{keystore_path}}
```

Comment:
one way v 2 way?

Reply:
oh sorry its a comment for me. If its only 1way tls then you dont need the keystore here just truststore. right?